src/Common/src/Interop/Windows/kernel32/Interop.GetFileAttributesEx.cs

```diff
@@ -68,10 +67,8 @@ internal unsafe struct WIN32_FIND_DATA
             internal uint nFileSizeLow;
             internal uint dwReserved0;
             internal uint dwReserved1;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 260)]
-            internal string cFileName;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 14)]
-            internal string cAlternateFileName;
+            internal FixedString.Size260 cFileName;
+            internal FixedString.Size14 cAlternateFileName;
```

Comment:
> discoverable

Interop is a low-level thing, and when I look at an interop signature, I want to a) map as 1:1 as possible with the underlying C data types, and b) be written in a way that's as easy to verify as possible.  Abstracting it in this manner means I need to understand the implementation of those structs to confirm it, and means it's much more difficult for me to prove it's correct.

> making mistakes

This may help with some possible mistakes, but it introduces the possibility for others, e.g. every call to Value allocating a new string.

And FWIW, I don't see how these structs help to eliminate mistakes.  You can do the exact same things as helpers.  They could even be instance methods on the interop struct, and it would still be better in my mind than having these structs.